src/main/java/de/srendi/advancedperipherals/common/setup/CCRegistration.java

```diff
@@ -50,7 +50,7 @@ public class CCRegistration {
 
     public static void register() {
         IntegrationPeripheralProvider.load();
-        ComputerCraftAPI.registerAPIFactory(unused -> new GlobalAPHelperAPI());
+        ComputerCraftAPI.registerAPIFactory(unused -> new GlobalVersionsLuaAPI());
```

Comment:
since it's not related to computer environment, we can also use singleton for it.

Reply:
I agree